### DISCUSSION TITLE
Window functions run much longer over joined tables than a single table. Is it a bug?

### DISCUSSION THREAD

**Discussion by vadik299** [duckdb/duckdb#13018] [Q&A]
I have a large query with many window functions and a join of 5 tables in the from clause. It runs for ~50 sec. If instead I prebuild a single table containing everything, same functions run for 15 sec. The table itself is created within ~1 sec time and contains about 350k records. According to query plan the same thing is done internally, however much slower. Any thoughts? I tried it on a simpler tables using public data samples, but with those it actually works as expected. I can't rewrite every query, is there any setting I can try to change this behavior? Many thanks for any ideas!

Basically this takes a long time:
select sum(field1) over (...) 
from table1 join table2 on ...

while this is much faster:

create table j as select * from table1 join table2 on ... ;

select sum(field1) over (...) 
from j;

**Comment by soerenwolfers** [comment]
If you have a MWE, this should be reported as an issue.
